src/contribution-guide/contributing.md

```diff
@@ -0,0 +1,25 @@
+# Contribute to Kubewarden docs
+
+_Kubewarden welcomes suggestions for improvement from all contributors, new and experienced!_
+
+You can contribute to Kubewarden documentation by
+
+- Improving existing content
+- Creating new content
+- (_Upcoming_) Helping with translation of the documentation
+
+## Getting Started
+
+Everybody is welcome to suggest changes by submitting a pull request
+or report a bug/typo with the help of a GitHub issue to the
+[`kubewarden/docs` repository](https://github.com/kubewarden/docs).
+Knowledge of [git](https://git-scm.com/) and [GitHub](https://lab.github.com/) is
+an essential pre-requisite to getting started.
+
+To get involved with the documentation:
+- Familiarize yourself with the [documentation repository](https://github.com/kubewarden/docs)
+and the [static site generator](https://rust-lang.github.io/mdBook/).
+- mdBook uses the [CommonMark spec](https://commonmark.org/) of Markdown.
+Ensure you're familiar with the Markdown flavour before you begin.
+- Understand how to [suggest an improvement]() and [open a pull request]().
```

Comment:
Are this links going to be filled later on?

Reply:
Hello Rafael, yes this folder will contain documents for the contributor guide. This is the front page. I'm working on it section-by-section.